site/themes/arangodb-docs-theme/static/css/theme.css

```diff
@@ -144,6 +144,44 @@ button, input, optgroup, select, textarea {
     margin-bottom: 15px;
 }
 
+.button {
+    background: transparent;
+}
+
+.button__wrapper {
+    border-radius: 4px;
+    text-align: center;
+    text-decoration: none;
+    border: none;
+    cursor: pointer;
+    width: fit-content;
+    height: 2rem;
```

Comment:
Let's use `px` instead of `rem`.. Ideally they should be used only for `fonts` .. But having `px` is much more consistent